string figure
float64 scaleX
float64 scaleY
float64 scaleZ
float64 offsetX
float64 offsetY
float64 offsetZ
---
bool success
string message